releasenotes/notes/nova-13.0-0c2497128e677cb2.yaml

```diff
@@ -0,0 +1,20 @@
+---
+prelude: >
+    This version introduces nova's new database,
+    nova api db.
+security:
```

Comment:
I was talking about the switch to qemu+ssh, I'm not sure what you mean by 'mentioning it would be insecure' though...